### PR TITLE
Fix/#102 and #99 issues

### PR DESCRIPTION
**Pull request by litvinets**
Fix for issues 102 and 99
1. The header (template and styles )is changing depending on the routing links (main/activities/config). 
2. Merged with Yura's changes.

### CODE REVIEW

File: src/app/header/header.component.html [hunk 1]
```diff
@@ -1,4 +1,4 @@
-<div class="header">
+<div class="header header_main" [ngClass]="(isMainPage$ | async) ? 'header_main' : 'header_not_main'">
```
Comment (dmitryy90) [inline src/app/header/header.component.html:1]: please use meaningful class names

**Review by dmitryy90 — CHANGES_REQUESTED** [review]

File: src/app/header/header.component.html (hunk 1 above)
Comment (litvinets) [reply src/app/header/header.component.html:1]: class="header_main_page"
class="header_not_main_page"

Is it gonna be ok?

Comment (litvinets) [reply src/app/header/header.component.html]: @dmitryy90 
class="header_main_page"
class="header_not_main_page"

Is it gonna be ok?

Comment (dmitryy90) [reply src/app/header/header.component.html]: header > main  | header > secondary
or any descriptive class name
not_main makes no sense to me

**Review by dmitryy90 — APPROVED** [review]

**Pull request merged**